### PR TITLE
SUS-3648 | remove ooyala-related code

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/SUS-3848

@Wikia/sus 
@Wikia/adeng 
@Wikia/x-wing 
@Wikia/mobile-apps

### CODE REVIEW

**Review by macbre — APPROVED** [review]
Red is my favourite colour :)

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 1]
```diff
@@ -129,7 +129,6 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		$vars[] = 'wgAdDriverF2BabDetectionCountries';
 		$vars[] = 'wgAdDriverF2InstartLogicRecoveryCountries';
 		$vars[] = 'wgAdDriverF2MEGAVideosCountries';
-		$vars[] = 'wgAdDriverF2OoyalaPosKeyValueCountries';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Revert it please. It's still in use and it shouldn't be removed

**Review by fraszczakszymon — CHANGES_REQUESTED** [review]
Revert instant global please

**Review by fraszczakszymon — APPROVED** [review]

**Review by fraszczakszymon — APPROVED** [review]

**Review by macbre — APPROVED** [review]

**Review by mrmike — APPROVED** [review]
LGTM from apps

**Pull request merged**